authors/apps/authentication/tests/test_email_verification.py

```diff
@@ -0,0 +1,64 @@
+import os
+from django.urls import reverse
+from django.core import mail
+from django.conf import settings
+from rest_framework.test import APIClient
+from rest_framework.test import APITestCase
+from rest_framework.views import status
+from ..utils import send_link
+
+
+class TestEmailVerification(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.signup_url = reverse('authentication:auth-register')
+        self.login_url = reverse('authentication:auth-login')
+        self.verify_url = os.getenv('EXPIRED_LINK')
```

Comment:
If you only need EXPIRED_LINK for testing, then no need to put in the environment. Just have it in the test.

Reply:
The instance variable name should also reflect that link is expired.

Reply:
I have to put it there since the environment changes when the app is deployed